utils/__init__.py

```diff
@@ -0,0 +1,2 @@
+# coding=utf-8
+"""Tools used for generating config files for camayoc."""
```

Comment:
I think we can place the gen_config.py script in a directory called scripts or tools. It does not need to be a Python package since the files we can add there can be considered standalone scripts.